Documentation/configuring-hive-metastore.md

```diff
@@ -35,20 +35,20 @@ Use [metastore-storage.yaml][metastore-storage-config] as a template and adjust
 By default to make installation easier Metering configures Hive to use an embedded Java database called [Derby](https://db.apache.org/derby/#What+is+Apache+Derby%3F), however this is unsuited for larger environments or metering installations with a lot of reports and metrics being collected.
 Currently two alternative options are available, MySQL and PostgreSQL, both of which have been tested with the metering-operator.
 
-There are 4 configuration options you can use to control the database used by Hive metastore: `url` , `driver` , `username` , and `password`.
+There are 4 configuration options you can use to control the database used by Hive metastore: `url` , `driver` , `username` , `secretName` and `password`.
```

Comment:
if you use `secretName` then `username` and `password` doesn't apply, right?

Reply:
@emoss but if the username & password parameter is still valid, then the beginning should be updated to `5`... or to whatever number of parameters that are accepted.

Reply:
For Mysql it'll be 4, with Postgres 5, but I will clarify